core/src/test/java/de/evoila/cf/broker/service/impl/AsyncDeploymentServiceImplTest/AsyncCreateInstanceTest.java

```diff
@@ -10,20 +11,24 @@
 
 class AsyncCreateInstanceTest extends BaseTest {
 
-    private void whensForSyncCreateInstanceThrowsException(JobProgress returnOfFailJob) throws ServiceBrokerException
-    {
+    private static final String JOB_PROGRESS_DESCRIPTION = "Creating service..";
+
+
+    private void createInstanceThrowsException(Exception expectedException) throws ServiceBrokerException{
         mockSuccessfulStartJob(JobProgress.PROVISION);
-        ServiceBrokerException expectedException = new ServiceBrokerException("Test");
         when(deploymentService.syncCreateInstance(serviceInstance, parameters, plan, platformService))
                 .thenThrow(expectedException);
-        when(startedJob.getId())
-                .thenReturn(JOB_PROGRESS_ID);
-        when(jobProgressService.failJob(eq(JOB_PROGRESS_ID), anyString()))
-                .thenReturn(returnOfFailJob);
     }
 
-    private void whensForSyncCreateInstanceSucceeds(JobProgress returnOfSucceedProgress) throws ServiceBrokerException
-    {
+    private void whensForSyncCreateInstanceThrowsServiceBrokerException() throws ServiceBrokerException {
```

Comment:
Are the methods whensForSyncCreateInstanceThrowsServiceBrokerException and whensForSyncCreateInstanceThrowsRuntimeException really needed? Why not call createInstanceThrowsException directly?

Reply:
fixed